cascalog-more-taps/test/cascalog/more_taps_test.clj

```diff
@@ -40,6 +40,24 @@
             ((hfs-delimited tmp :delimiter "," :compress? true) ?a ?b ?c)) =>
         (produces [["Proin" "false" "3"]])))))
 
+(deftest delimited-compression-test
```

Comment:
Yeah, I just copied the format from the :compress? test. I guess if we set it to compress and then tried to read it with compression disabled we would expect a failure?